utils/convertStringToSlug.ts

```diff
@@ -1,4 +1,4 @@
-export default function convertToSlug (string) {
+export default function convertToSlug (string:string) {
   if (!string || typeof string !== 'string') {
     return string
   }
```

Comment:
Using `string` as a variable or parameter name is a bad idea in general -- would prefer we update this to use something like `text` or `name` or something relevant to the use cases that doesn't overlap with type names.